WeakAurasOptions/ActionOptions.lua

```diff
@@ -284,7 +284,8 @@ function OptionsPrivate.GetActionOptions(data)
         values = {
           UNITFRAME = L["Unit Frame"],
           NAMEPLATE = L["Nameplate"],
-          FRAMESELECTOR = L["Frame Selector"]
+          FRAMESELECTOR = L["Frame Selector"],
+          PARENTFRAME = L["Parent Frame"]
         },
```

Comment:
(not your fault, but) I think we should use `OptionsPrivate.Private.glow_frame_types` here
```suggestion
        values = OptionsPrivate.Private.glow_frame_types,
```